nightingale_dispatcher/src/nightingale_dispatcher/task.py

```diff
@@ -1,6 +1,23 @@
 #!/usr/bin/env python3
+import rospy
+from nightingale_ros_bridge import BridgeConfig, RobotStatus, UserInputs
+from nightingale_msgs.srv import InterfaceCall
 
 
 class Task:
-    ERROR = -1
-    SUCCESS = 0
+    def __init__(self):
+        self.ERROR = -1
+        self.SUCCESS = 0
+        self.interface_comms_service_name = BridgeConfig.UPDATE_UI_SERVICE
+
+    def interface_comms_client(self, robot_state):
```

Comment:
Can you make this function name more descriptive. Something like `wait_user_input()` or `get_user_input()` is a bit more clear about what the function is actually doing

Reply:
The interface communication doesn't always get user input though and sometimes is just used to update the state of the robot for the tablet to take action on it. Would "send robot state" be better?